### DISCUSSION TITLE
Group Assignment #3: Choropleth map

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#59] [Assignments]
# Group Assignment #3 Choropleth map

- Group name
- Group members
- Title of the map
- Description of the map
- Link to choropleth map

**Comment by kimjee8955** [comment]
- Datart
- Annmarie Cong, Eustina Kim, Michelle Lee
- Title: Number of Art Pieces by Region
- Description: shows the number of public art pieces in each neighborhood of LA County from our dataset.
- Link: https://kimjee8955.github.io/Choropleth/index.html

**Comment by ryanorlando** [comment]
- Team Data
- Ryan Orlando, Ainsleigh Douglas, Sebastien Dills
- Title: CO2 Emissions per Unit of GDP
- Description: Shows the quantity of carbon emissions per unit of GDP for countries around the world
- Link: https://ryanorlando.github.io/Dh151/Week7/index.html

**Comment by alekhyavittalam** [comment]
- State-of-the-Climate
- Natasha Grace, Alekhya Vittalam and Zona Liao
- Title: Impact of Outdoor Air Pollution
- Description: shows the percentage of air pollution in each country in the year 2016
- Link: https://alekhyavittalam.github.io/state-of-the-climate/Week7Project/map.html

**Comment by lisa58** [comment]
- Fantastic Four
- Labiba Alam, Haiqi, Adriana Romero, Lynn Tieu
- Title: Number of Anti-Asian Incidents in the U.S. by State
- Description: anti-Asian hate incidents from March 2020 to March 2021 as reported by The New York Times
- Link: https://lynnt2.github.io/FantasticFour/GroupAssignments/Assignment3/index.html

**Comment by jmtanenbaum** [comment]
Group: Environmental Racism
Julia Tanenbaum, Hannah Lien, Lisa Lou 
Title: Map of Groundwater and Air Pollution near California Prisons, Jails, and Detention Facilities 
Description: This map uses data from Cal Enviro Screen and Covid Behind Bars to juxtapose environmental pollution with prison populations and locations in California. 
link: https://jmtanenbaum.github.io/DHEnviroIncarceration/index.html

**Comment by frida-barragan** [comment]
- Health Mappers
- Frida & Kate 
- Title: Mental Health Crisis and Services in Los Angeles County
- Description : Raw counts and data points of the mental health services in LA county by zip code.
- Link: https://frida-barragan.github.io/DH151/joindata/index.html

**Comment by justisg** [comment]
- The Musical Maps
- Daniel Liu, Kai Watanabe, Justis Guardia
- Music, Concerts, and COVID-19
- Shows the number of concerts organized by location and amount in 2019 and 2020
- https://justisg.github.io/TheMusicalMaps/Website/index.html

**Comment by hoz-map** [comment]
- Rethink Recycle
- Alice Lu, Emily Chau, Hozumi Oshida
- Mismanaged Plastic in the World
- Shows the number of mismanaged plastic waste generated in 2010
- https://hoz-map.github.io/Rethink-Recycling/Week7%20final/index.html

**Comment by alexthegreat1** [comment]
- cARTographers
- Kai Colorado, Alex Haddad
- Mapping U.S. Art Education
- Maps and shows relationships between per-state arts education policies and other factors, such as standardized assessment scores.
- https://kaicolorado.github.io/DH151-Group/index.html

Note: When any standardized scores layers are selected, states are colored as a heatmap from blue to red based on their average score. These colors by attribute can only be seen when an arts education policy is not also selected. This is to prevent too many colors from overlapping.

**Comment by MaryJaneHowland** [comment]
Happy Tweet
Ivan Ma, MJ Howland, Josephine Meyer-Hogan
Happiness Map
Shows happiness scores in each city 
https://ivanma9.github.io/HappyTweet/

Note: we are currently working to make the map better so that the map is more meaningful. You can currently view the map by clicking on happiness map.

**Comment by derekwang99** [comment]
* Ocean Pollution Posse
* Eli Aranita, Julia Wood, Derek Wang
* Plastic Pollution World Choropleth
* Our Map shows the amount of mismanaged plastic has polluted from 128 Countries in 2010
* https://eliaranita1.github.io/project1/Group_Assignment_3/index.html
